plugin-server/src/worker/vm/extensions/jobs.ts

```diff
@@ -49,7 +67,9 @@ export function createJobs(server: Hub, pluginConfig: PluginConfig): Jobs {
             }
             server.statsd?.increment('job_enqueue_attempt')
             await server.enqueuePluginJob(job)
+            jobsEnqueuedCounter.inc()
```

Comment:
I've got a feeling that this is run in Piscina and therefore you won't see the keys in `/_metrics` as it stands, or is that not the case?

Reply:
Arf, you are right!
Until we get the piscina worker stuff sorted out, could we use the number of kafka messages written to the `jobs` topic?

Reply:
Indeed, they stay at zero due to the multiprocess. `prom-client`'s multi-process support relies on the `cluster` lib and I'm not sure whether it would play nice with piscina.

Removing them and going with the kakfa metric for now.